File: esm/debug_json.php
<?php
$file = __DIR__ . '/artwork_data.json';
if (!file_exists($file)) {
    echo "File not found\n";
    exit;
}

$content = file_get_contents($file);
$data = json_decode($content, true);

if (json_last_error() !== JSON_ERROR_NONE) {
    echo "JSON Error: " . json_last_error_msg() . "\n";
    // Try to find where it breaks?
    // simple heuristic: find where parsing stops?
} else {
    echo "JSON is valid.\n";
    echo "Count: " . count($data) . "\n";
    foreach ($data as $item) {
        if (isset($item['title'])) {
            echo "Title: " . $item['title'] . "\n";
            if (isset($item['description'])) {
                // Check if description looks suspicious (contains { or [ or ;)
                if (strpos($item['description'], '{') !== false || strpos($item['description'], 'function') !== false) {
                    echo "  WARNING: Suspicious description in " . $item['title'] . "\n";
                    echo "  Desc start: " . substr($item['description'], 0, 50) . "...\n";
                }
            }
        }
    }
}
?>

File: esm/debug_json.py
import json
import os
import sys

file_path = r'c:\sandbox\esm\artwork_data.json'

if not os.path.exists(file_path):
    print("File not found")
    sys.exit(1)

try:
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()
        data = json.loads(content)
        
    print("JSON is valid.")
    print(f"Count: {len(data)}")
    
    for item in data:
        title = item.get('title', 'NO TITLE')
        desc = item.get('description', '')
        if '{' in desc or 'function' in desc or '<?php' in desc:
            print(f"WARNING: Suspicious description in '{title}'")
            print(f"  Desc start: {desc[:100]}...")
            
except json.JSONDecodeError as e:
    print(f"JSON Error: {e}")
except Exception as e:
    print(f"Error: {e}")
